### PR TITLE
chore: README minor formatting changes

### PR DESCRIPTION
**Pull request by miguelsorianod**
## Description
Updat README with some minor formatting changes. Those include fixing formatting issues, making the formatting consistent throughout the file, ...

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] All PR comments are resolved either by addressing them or creating follow up tasks
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has been created for changes required on the client side

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,149 +1,199 @@
...
+       ```
+       ```
+       # List all the tables
+       serviceapitests# \dt
+                          List of relations
+       Schema |        Name        | Type  |       Owner
+       --------+--------------------+-------+-------------------
+       public | clusters           | table | kas_fleet_manager
+       public | connector_clusters | table | kas_fleet_manager
+       public | connectors         | table | kas_fleet_manager
```
Comment (machi1990) [inline README.md:167]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -157,28 +207,37 @@ curl -v -XGET -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/
...
 
-# Here c7ndprea8ueq4mmrm3c0 is the kafka instance ID.
-```
+* The admin-server API is used for managing topics, acls, and consumer groups.
+  The API specification can be found [here](https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/src/main/resources/openapi-specs/kafka-admin-rest.yaml)
+* To get the API endpoint, use the following command
+    ```
+    # Here c7ndprea8ueq4mmrm3c0 is the kafka instance ID
+    oc get routes -n kafka-c7ndprea8ueq4mmrm3c0 | grep admin-server
+    ```
```
Comment (machi1990) [inline README.md:237]: This is not needed anymore. 
The admin-serve endpoint is returned as part of the Kafka GET/LIST endpoints as soon as the Kafka is ready - https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/acddfc4de0fa99fc22a268f1a3e0fdf135cb7745/openapi/kas-fleet-manager.yaml#L983

File: README.md (hunk 1 above)
Comment (miguelsorianod) [reply README.md:167]: This PR is about formatting changes but I will change it anyway :+1:

File: README.md (hunk 2 above)
Comment (miguelsorianod) [reply README.md:237]: This PR is about formatting changes but I will change it anyway :+1:

File: README.md (hunk 1 above)
Comment (machi1990) [reply README.md]: Thanks.

File: README.md (hunk 2 above)
Comment (machi1990) [reply README.md]: Thanks

Comment (miguelsorianod) [reply README.md]: Changed

File: README.md (hunk 1 above)
Comment (miguelsorianod) [reply README.md]: Changed

**Review by machi1990 — APPROVED** [review]

**Pull request merged**